.github/workflows/hds-demo-preview.yml

```diff
@@ -0,0 +1,123 @@
+name: hds-demo-preview
+
+on:
+  pull_request:
+    branches:
+      - development
```

Comment:
I'm just wondering that should we also make these demos for the PR's targeting release-x.x.x -branches?

Reply:
added 'release-*'